src/edge_op.cpp

```diff
@@ -502,14 +502,14 @@ void Manifold::Impl::CollapseEdge(const int edge, std::vector<int>& edges) {
           // Don't collapse if the edges separating the faces are not colinear
           // (can happen when the two faces are coplanar).
           if (CCW(projection * pOld, projection * pLast, projection * pNew,
-                  precision_) != 0)
+                  tolerance_) != 0)
```

Comment:
I'm thinking about these CCW checks. I think some should probably be using `epsilon` instead of `tolerance`, but not sure which is which. E.g. for the next one which checks for if collapsing will cause inverted triangle, we should probably be using `epsilon`, but not sure if that is the only one required.